ytypes/gnmi_test.go

```diff
@@ -496,6 +496,48 @@ func TestUnmarshalSetRequest(t *testing.T) {
 			}},
 		},
 		wantErr: true,
+	}, {
+		desc: "mix of an error and a non-error update with best-effort flag",
```

Comment:
For testing I recommend using a complete set-up but testing **two** errors in each of 1) delete 2) replace 3) update and then checking that all six errors are present. This way you get coverage of all six modifications in the handling logic.

Reply:
I think I understand - I'll upload a new version in a few, let me know if I've misunderstood.